.devtools/fop_names_rules.neon

```diff
@@ -0,0 +1,11 @@
+services:
+  -
+    class: FOP\Console\DevTools\PhpStanCustomRule
+    tags:
+      - phpstan.rules.rule
+  -
+    class: FOP\Console\Tests\Validator\FOPCommandFormatsValidator
+  -
+    class: FOP\Console\Tests\Validator\PhpStanNamesConsistencyService
```

Comment:
This class (PhpStanNamesConsistencyService) is injected (auto wired) in the phpstan rule (PhpStanCustomRule).
It's first argument is a file path declared below.
The second argument (FOPCommandFormatsValidator) is not declared, it's autowired.